### PR TITLE
Fix the issue that trace_sql_parameter argument is invalid for oracle db plugin

### PR DESCRIPTION
**Pull request by raybi-asus**
1. Create a new class named OraclePrepareStatementSetterInstrumentation which is extend from OraclePrepareStatementInstrumentation
2. Add some lines in PreparedStatementExecuteMethodsInterceptor to process the sql parameter
3. remove the snapshot suffix from pom.xml, change the compiler from 1.6 to 8
4. remove the invalid content from travis.yml
5. Add OraclePrepareStatementSetterInstrumentation in the bottom of skywalking-plugin.def

### CODE REVIEW

**Comment by raybi-asus** [comment]
local build_20201201150958

Actually the maven command ran successfully in my local env...Not sure what happened

File: oracle-10.x-plugin/src/main/java/io/skywalking/apm/plugin/jdbc/oracle/PreparedStatementExecuteMethodsInterceptor.java [hunk 1]
```diff
@@ -20,33 +20,52 @@
...
 
 /**
  * @author zhang xin
  */
 public class PreparedStatementExecuteMethodsInterceptor implements InstanceMethodsAroundInterceptor {
+  
+    public static final StringTag SQL_PARAMETERS = new StringTag("db.sql.parameters");
 
+    /**
+     * 2020-12-01 ray_bi modified
```
Comment (wu-sheng) [inline oracle-10.x-plugin/src/main/java/io/skywalking/apm/plugin/jdbc/oracle/PreparedStatementExecuteMethodsInterceptor.java:43]: Please don't add your name. Changelog has been auto-added by git .

Comment (raybi-asus) [reply oracle-10.x-plugin/src/main/java/io/skywalking/apm/plugin/jdbc/oracle/PreparedStatementExecuteMethodsInterceptor.java:43]: Ok, i will remove it asap...

File: .travis.yml [hunk 2]
```diff
@@ -4,13 +4,12 @@ language: java
 before_install:
   - mkdir ci-dependencies
   - cd ci-dependencies
-  - curl -O https://openskywalking.github.io/skywalking-ci-assist/jars/ojdbc14-10.2.0.4.0.jar
+  - curl -O https://github.com/SkyAPM/ci-assist/blob/master/jars/ojdbc14-10.2.0.4.0.jar
```
Comment (wu-sheng) [inline .travis.yml:7]: https://github.com/SkyAPM/ci-assist/blob/master/jars/ojdbc14-10.2.0.4.0.jar?raw=true

This is the real download URL, if you want to download them from GitHub

Comment (wu-sheng) [reply .travis.yml:7]: Or this, https://skyapm.github.io/ci-assist/jars/ojdbc14-10.2.0.4.0.jar

File: .travis.yml [hunk 3]
```diff
@@ -4,13 +4,12 @@ language: java
...
   - mvn install:install-file  -Dfile=ojdbc14-10.2.0.4.0.jar  -DgroupId=com.oracle  -DartifactId=ojdbc14 -Dversion=10.2.0.4.0 -Dpackaging=jar
-  - curl -O https://openskywalking.github.io/skywalking-ci-assist/jars/resin-4.0.41.jar
+  - curl -O https://github.com/SkyAPM/ci-assist/blob/master/jars/resin-4.0.41.jar
```
Comment (wu-sheng) [inline .travis.yml:9]: Same pattern here.

Comment (raybi-asus) [reply .travis.yml]: As you mentioned , I changed the uri on both ojdbc and resin.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**